jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/IPC.java

```diff
@@ -98,6 +109,9 @@ public class IPC {
 		}
 		isZOS = tempIsZos;
 		isWindows = tempIsWindows;
+
+		useFileLockWatchdog = !(isZOS
```

Comment:
Is it intended that -Dcom.ibm.tools.attach.disableFileLockWatchdog=false can't overwrite the default behaviour on z/OS? Is there any use to have it work just in case it needs to be enabled for diagnostic purposes?